Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/TrackSelector.cpp

```diff
@@ -43,7 +43,7 @@ ActsExamples::ProcessCode ActsExamples::TrackSelector::execute(
     const bool validCharge = validNeutral or validCharged;
     return validCharge and
            within(trk.transverseMomentum(), m_cfg.ptMin, m_cfg.ptMax) and
-           within(std::abs(theta), m_cfg.absEtaMin, m_cfg.absEtaMax) and
```

Comment:
Arguably, this should be a separate PR.

Reply:
yeah I fear I bundled a few things together here to make this work :smile:

Reply:
sourced this out to https://github.com/acts-project/acts/pull/1615